settings.py

```diff
@@ -7,26 +7,17 @@
 #     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
 #     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
 
-# USER_AGENT = get_random_agent()
-
 BOT_NAME = 'scraping'
 
 SPIDER_MODULES = ['scraping.spiders']
 NEWSPIDER_MODULE = 'scraping.spiders'
 
 # Crawl responsibly by identifying yourself (and your website) on the user-agent
-# USER_AGENT = 'scraping (+http://www.yourdomain.com)'
+# USER_AGENT = 'newone (+http://www.yourdomain.com)'
```

Comment:
Why changed this ? Its a comment, changing it will do nothing.

Reply:
new project creat kia tha ya us key setting.py file thi jis ma project name change kia tha